src/assets/translations/en/main.json

```diff
@@ -239,7 +239,8 @@
     "walletBalance": "Wallet Balance",
     "earnBalance": "Earn Balance",
     "earnBody": "Earn passive income by staking your assets or depositing them into a DeFi strategy.",
-    "earnCopy": "Click here to Earn %{apr}%+ APR* by farming your ETH-FOX liquidity pool tokens",
+    "clickHereToEarn": "Click here to Earn ",
+    "byFarming": " APR* by farming your ETH-FOX liquidity pool tokens",
```

Comment:
i've been burnt by this before - it's generally not a great idea to break up translations grammatically, but it's probably fine